src/cmd/metadata.rs

```diff
@@ -56,12 +56,12 @@ impl GenerateMetadataCommand {
         util::assert_channel()?;
 
         let cargo_meta = &self.crate_metadata.cargo_meta;
```

Comment:
This binding is now redundant, can just move to usage site now (`Workspace::new`). A further refactoring might be to just have `Workspace::new` accept `&CrateMetadata`.